OpenRA.Mods.Common/Traits/Player/EnemyWatcher.cs

```diff
@@ -101,8 +101,7 @@ public void Tick(Actor self)
 		void Announce(Actor self, TraitPair<AnnounceOnSeen> announce)
 		{
 			// Audio notification
-			if (self.World.LocalPlayer != null)
```

Comment:
To fix the issue Mailaender noted in his last comment. You'd need to wrap the whole body of this method beneath that check.